components/base/agilex/limo_base.go

```diff
@@ -129,6 +132,12 @@ func createLimoBase(ctx context.Context, _ resource.Dependencies, conf resource.
 		rightAngleScale:    1.64,
 	}
 
+	geometries, err := kinematicbase.CollisionGeometry(conf.Frame)
+	if err != nil {
+		logger.Warnf("base %v not configured with a geometry, use caution if using motion service", lb.Name())
```

Comment:
more information on what geometry does, something to tell the use that you're not planning with collisions in mind if I'm interpreting this error message right.

Reply:
Sounds good. Are you ok with this warning message in general?  I added it to every implementation including `MyBase`, and I wasn't sure if that was kosher

Reply:
That's fine. That's just an example code for someone to look at when implementing a module.